package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "wdfn-viz",
-  "version": "0.1.11-snapshot",
+  "version": "1.0.0-release",
```

Comment:
Set this to 1.0.0-snapshot until the ticket is tested and accepted.

Reply:
Will do.

Reply:
Oh, so then the change log should still say unreleased, and not '## [1.0.0][]' ?

Reply:
Yes.